docs/vf-setup.md

```diff
@@ -1,68 +1,83 @@
 # Setting up Virtual Functions
 
 The SR-IOV Network Device Plugin requires SR-IOV virtual functions (VFs) to be created on the host ahead of startup. The device plugin does not manage the creation of virtual functions, and it does not automatically update when there is a change in the virtual functions. Each time the number of VFs or the driver used by the VF change the SR-IOV Network Device Plugin needs to be restarted.
-This guide gives an overview for setting up SR-IOV virtual functions on a linux host for various NICs. 
+This guide gives an overview for setting up SR-IOV virtual functions on a linux host for various NICs.
 
-  * [Intel](#intel)
-  * [Mellanox](#mellanox)
+* [Intel](#intel)
+* [Mellanox](#mellanox)
 
 ## Intel
+
 The below works on Intel SR-IOV enabled cards including those from the Gigabit Ethernet, X710 and X722 families of Network Interface Cards. Note not all Gigabit Ethernet NICs are enabled with SR-IOV.
 
 Cards from the following families can be enabled through the sysfs endpoint as described below:
+
 * **Intel i40e Driver**
-	* Intel® Ethernet Controller X722 
-	* Intel® Ethernet Controller X710
-* **Intel ixgbe Driver** 
-	* Intel® Ethernet Controller 82599
-	* Intel® Ethernet Controller X520
-	* Intel® Ethernet Controller X540
-	* Intel® Ethernet Controller x550
-	* Intel® Ethernet Controller X552
-	* Intel® Ethernet Controller X553
+  
+  * Intel® Ethernet Network Adapter X722
```

Comment:
can you please also add a list for the E810 supported nics?

Reply:
Added E810 NICs and also removed line about support for k8s <1.15